src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -20,8 +20,8 @@ public static class Parser
 
         private static readonly Regex[] ReportMovieTitleRegex = new[]
         {
-            //Some german or french tracker formats (missing year, ...) (Only applies to german and French/TrueFrench releases) - see ParserFixture for examples and tests
-            new Regex(@"^(?<title>(?![(\[]).+?)((\W|_))(" + EditionRegex + @".{1,3})?(?:(?<!(19|20)\d{2}.*?)(German|French|TrueFrench))(.+?)(?=((19|20)\d{2}|$))(?<year>(19|20)\d{2}(?!p|i|\d+|\]|\W\d+))?(\W+|_|$)(?!\\)", RegexOptions.IgnoreCase | RegexOptions.Compiled),
+            //Some german or french tracker formats (missing year, ...) (Only applies to german and TrueFrench releases) - see ParserFixture for examples and tests - french removed as it broke all movies w/ french titles
+            new Regex(@"^(?<title>(?![(\[]).+?)((\W|_))(" + EditionRegex + @".{1,3})?(?:(?<!(19|20)\d{2}.*?)(German|TrueFrench))(.+?)(?=((19|20)\d{2}|$))(?<year>(19|20)\d{2}(?!p|i|\d+|\]|\W\d+))?(\W+|_|$)(?!\\)", RegexOptions.IgnoreCase | RegexOptions.Compiled),
```

Comment:
Isn't German going to have the same issue?

Reply:
yes, but these releases are "normal" for the german scene; however none of the test cases have the year in them.

there also seem to be more movies w/ french in the title than german

can we rip this logic out completely?  This was implemented when lenient parsing was ripped out in 2020 https://github.com/Radarr/Radarr/commit/51de8f16fb404f5adece0f9447cfc44dd13cc7ae

We don't support releases without years, so I'm not sure why we have test cases and regex to support movies without years?

My thoughts are to rip out this regex line and any related failing tests

Reply:
I wouldn't rip it out completely.

Reply:
I think it's fine as is (assuming we're good with the regression)


- https://www.themoviedb.org/search/movie?query=german  389 movies
- https://www.themoviedb.org/search/movie?query=french 328 movies
- https://www.themoviedb.org/search/movie?query=truefrench 0 movies

perhaps we leave in only truefrench?